devel/ci/integration/tests/test_bodhi.py

```diff
@@ -386,11 +386,10 @@ def test_get_update_view(bodhi_container, db_container):
         if update_info['request']:
             assert update_info['request'] in http_response.text
         if update_info['autokarma']:
-            assert "Stable by Karma" in http_response.text
```

Comment:
Can you re-add the right assert here?

Reply:
Should I also include it in the `else` statement?

Reply:
I mean `The update will be automatically pushed to stable when karma reaches` like in `test_updates.py`. The else statement is fine (but you should remove the extra whitespace at the start of the line, check the indentation).